### DISCUSSION TITLE
Missing report "GET_FBA_FULFILLMENT_INVENTORY_EXPIRATION_DATE_DATA"

### DISCUSSION THREAD

**Discussion by AH1622** [abuzuhri/Amazon-SP-API-CSharp#514] [Q&A] [answered]
Hi,

Is the a reason why "GET_FBA_FULFILLMENT_INVENTORY_EXPIRATION_DATE_DATA" report missing in your library? I can see that this report exists in "Feeds, Reports and Notifications" section in "Amazon Fulfillment role chapter" of SP-API documentation.

Thank you in advance!

**Comment by ProNotion** [comment]
Interesting - I wasn't aware of that report but would find that incredibly useful since there appears to be no other way to see this information once a shipment has been sent. I can't find an example of what the report might look like though.

**Reply by AH1622** [reply]
Maybe you could recommend me something on the following issue?  I need to find a way to get the information on whether a product has an expiration date. Specifically,  if I want to send a list of SKUs for the shipment, which I created via API request, I need to figure out if Amazon requires for me to provide an expiration date for each of the  SKUs that I specified in the shipment list.

**Reply by AH1622** [reply]
The message I got from Amazon: "The report GET_FBA_FULFILLMENT_INVENTORY_EXPIRATION_DATE_DATA is not a report that Amazon has available to the public anymore. This is an error in the documentation related to roles, and I will make a submission to request it's removal."

**Reply by ProNotion** [reply]
That's disappointing

**Comment by AH1622** [comment, marked as answer]
The message I got from Amazon: "The report GET_FBA_FULFILLMENT_INVENTORY_EXPIRATION_DATE_DATA is not a report that Amazon has available to the public anymore. This is an error in the documentation related to roles, and I will make a submission to request it's removal."
